src/libraries/Microsoft.Extensions.Hosting/src/HostingHostBuilderExtensions.cs

```diff
@@ -236,7 +237,12 @@ public static IHostBuilder ConfigureDefaults(this IHostBuilder builder, string[]
                 }
 
                 logging.AddConfiguration(hostingContext.Configuration.GetSection("Logging"));
-                logging.AddConsole();
+#if NET6_0_OR_GREATER
+                if (!OperatingSystem.IsBrowser())
```

Comment:
Change the above code to use OperatingSystem.IsWindows instead of `RuntimeInformation.IsOSPlatform(OSPlatform.Windows)` so this event log provider can be trimmed.

Reply:
Good call. Missed that code